pages/solid-events.md

```diff
@@ -16,6 +16,7 @@ Solid Events provide an opportunity to meet others who are working on Solid or a
 
 | Date | Event | Organiser | Recordings, Slides and Notes |
 |------|-------|-----------|-----------------------------|
+|2023-06-28|[Solid World](https://www.eventbrite.co.uk/e/solid-world-2023-tickets-577704739427) |[Solid Team](https://solidproject.org/team) | Solid World will be held at 11:00 a.m. EDT/1700 CEST. We'll hear an update on CSS, an announcement of Inrupt's Solid hackathon winners, and presentations from Graphmetrix and Motius. Register on [Eventbrite](https://www.eventbrite.co.uk/e/solid-world-2023-tickets-577704739427).|
```

Comment:
@csarven Would this suffice to at least be able to promote this event for now? Alternatively we could also list Inrupt if you're more comfortable with that @hzbarcea? We can always update it back to Solid Team later if we get consencus on that.

```suggestion
|2023-06-28|[Solid World](https://www.eventbrite.co.uk/e/solid-world-2023-tickets-577704739427) |[Hadrian Zbarcea](https://github.com/hzbarcea) | Solid World will be held at 11:00 a.m. EDT/1700 CEST. We'll hear an update on CSS, an announcement of Inrupt's Solid hackathon winners, and presentations from Graphmetrix and Motius. Register on [Eventbrite](https://www.eventbrite.co.uk/e/solid-world-2023-tickets-577704739427).|
```

Reply:
In this case, any organiser besides Solid Team will suffice. It should not be changed once that's committed.

AFAICT, the first public announcement about this event predates ( https://matrix.to/#/!ICiXwJRUuUgyPLIEvn:gitter.im/$EO3PUU7C16H3CjOz40QTwbL5vErf6PZv2Vv7Bh1YTNQ ) Hadrian joining the Solid Team ( even as early as https://github.com/solid/team/blob/main/meetings/2023-05-17.md#proposed-solid-worldnewsletter-instructions ), and nothing on record (by the Solid Team) indicating that Hadrian is assigned to organise SW, .. let alone disclose/inform *any* information about the event to the Solid Team in a meeting or in chat.

[On a related note, people that are not even on the Solid Team have knowledge about the SW agenda and its publication ( https://forum.solidproject.org/t/solid-world-june-2023/6604 , https://web.archive.org/web/20230628165958/https://forum.solidproject.org/t/solid-world-june-2023/6604 ).]

That said, Solid events happen everywhere with different organisers, and solidproject.org can continue to acknowledge them. So, it is completely okay to acknowledge Hadrian or another entity, if they and the proposer of this PR agrees, as the organiser of this event.

Reply:
> Alternatively we could also list Inrupt if you're more comfortable with that @hzbarcea? We can always update it back to Solid Team later if we get consencus on that.
@Vinnl thanks for the proposal. I am perfectly fine with that. At this point, it's water under the bridge. The event took place a few hours ago and presenters had a lot of great material. I can see Solid getting more and more ground. And Jackson did a wonderful job. A recording will be available, as always.

I guess we will discuss this more in a different forum. I see an acknowledgement that nothing has changed (and a claim that this is a problem), I saw everybody acting in good faith, for the good of the community, yet an unprecedented action was taken to not publish the existence of the event, denying (to some extent) the solidproject.org's audience the opportunity to participate.

The risk/reward reasoning that went into this rejection eludes me, especially given the fact that steps we taken to address the issues @csarven raised. Oh, well...

Reply:
>yet an unprecedented action was taken to not publish the existence of the event, denying (to some extent) the solidproject.org's audience the opportunity to participate.

Can you explain how you've reached that interpretation?

I understood Vincent's response and change suggestion as a good indicator that my review and messages was clear enough about what's intended, but you might have misinterpreted.

Here are some of my messages:

>There is nothing on record that the Solid Team organised this event. Citations needed. Until then, please refrain from making that association.

>You may not be aware of deep and long running issues pertaining to open and transparency of Solid World.

>Solid events happen everywhere with different organisers, and solidproject.org can continue to acknowledge them.

I asked Chat GPT to paraphrase them and it came up with this:

>There is no record or documented evidence confirming the involvement of the Solid Team in arranging this event. It is necessary to provide citations or reliable sources to support any claims regarding their association. Until such verifiable evidence is presented, it is advisable to abstain from making any connection between the Solid Team and the event in question.

>It is conceivable that you are not cognizant of the deep-rooted and persistent issues surrounding the openness and transparency of Solid World.

>Solid events take place in various locations, organized by different individuals or groups, and solidproject.org can continue to recognize and acknowledge them.

Does that clarify my review and messages?

Reply:
"There is nothing on record that the Solid Team organised this event".  The Solid Team cannot and should not micromanage the details of each Solid World. It delegates that to nearby folks who trusted to do a useful job, and thanked for their time and  effort.  Solid World can still be said to be held under the auspices of the Solid Team.   @csarven You seem to want a world which is painfully bureaucratic. 

 And with phrases like "deep and long running issues pertaining to open and transparency" you sow distrust and suspicion where it is not helpful.

Reply:
Micromanagement is not intended. As discussed in Solid Teem meetings, what's being asked is the *minimal* transparency. Not every detail needs to be run  by the team, just some awareness.